src/wp-includes/class-wp-modules.php

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+ * Modules API: WP_Modules class.
+ *
+ * Native support for ES Modules and Import Maps.
+ *
+ * @package WordPress
+ * @subpackage Modules
+ */
+
+/**
+ * Core class used to register modules.
```

Comment:
Nit:
```suggestion
 * Core class used to register modules.
 *
 * @since 6.5.0
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5818/commits/212cdac16228c427c7fdd6f142f5d6127ef9ac78.